src/inverseRobot.js

```diff
@@ -25,6 +25,20 @@
  */
 function inverseRobot(robot) {
   // write code here
+  let inverseNewRobot = {};
+
+  for (const key in robot) {
+    const val = robot[key];
```

Comment:
do not shorten variable name
it is clear what it means, but take it into consideration for the next tasks

Reply:
I try not to do more than one return

Reply:
it is not a problem when you want to end function